src/Microsoft.DotNet.Interactive.Tests/Utility/DirectoryUtility.cs

```diff
@@ -61,9 +61,10 @@ public static class DirectoryUtility
 
         public static string GetPathToSrcDirectory()
         {
-            string pathAssembly = System.Reflection.Assembly.GetExecutingAssembly().Location;            
-            string folderAssembly = Path.GetDirectoryName(pathAssembly);            
-            return Path.GetFullPath(folderAssembly + @"..\..\..\..\..\..\src");            
+            string pathAssembly = System.Reflection.Assembly.GetExecutingAssembly().Location;
+            string folderAssembly = Path.GetDirectoryName(pathAssembly);
+            var ps = Path.DirectorySeparatorChar;
+            return Path.GetFullPath(folderAssembly + $"..{ps}..{ps}..{ps}..{ps}..{ps}..{ps}src");            
```

Comment:
This is safer with
```csharp
return System.IO.Path.Combine(folderAssembly ,"..","..","..","..","..","..","src");
```

Reply:
Yes, I agree. This looks much better than the string-interpolations.

Reply:
it had to put this to make the 3 related tests succeed:
count the dots :)

return Path.Combine(folderAssembly, "..", "..", "..", "..", "..", "src");

Reply:
We got rid of the whole method here: https://github.com/dotnet/interactive/pull/1176/commits/433e8f6991745e7a7ffe7ba4d388de27f4fc506c